app/lib/steps/helper.rb

```diff
@@ -49,14 +49,24 @@ def step_groups_for(session_data)
       end
 
       def all_sections
-        [CaseDetailsSection,
-         ApplicantDetailsSection,
-         IncomeSection,
-         CapitalSection,
-         PartnerSection,
-         PartnerIncomeSection,
-         PartnerCapitalSection,
-         PropertySection]
+        if FeatureFlags.enabled?(:household_section)
+          [CaseDetailsSection,
+           ApplicantDetailsSection,
+           IncomeSection,
+           PartnerIncomeSection,
+           OutgoingsSection,
+           CapitalSection,
+           PropertySection]
+        else
+          [CaseDetailsSection,
+           ApplicantDetailsSection,
+           IncomeSection,
+           CapitalSection,
+           PartnerSection,
+           PartnerIncomeSection,
+           PartnerCapitalSection,
+           PropertySection]
```

Comment:
PropertySection is always empty when the feature flag is disabled, so this can be removed.